runzi/automation/scaling/toshi_api/toshi_api.py

```diff
@@ -62,7 +62,7 @@ def get_general_task_subtasks(self, id):
                   title
                   description
                   created
-                  swept_arguments
+                  #swept_arguments
```

Comment:
@chrisdicaprio is query this causing an API error? If so we should fix it there as it will cause trouble in TUI. If you have a GT ID that does this can you pls raise a ticket on toshiAPI?